mondoo.ps1

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
+Write-Host -ForegroundColor Red "Mondoo binary is deprecated. Use cnspec."
```

Comment:
@misterpantz any chance we have some docs talking about this somewhere that we could link folks to?

Reply:
What if we take care of it right in the message? "Mondoo Client is deprecated. Instead, use [cnspec](https://mondoo.com/docs/cnspec/cnspec-about/), Mondoo's cloud-native security scanner and CLI."